parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -161,8 +170,10 @@ async def entry_info(self, path: FsPath) -> dict:
         self.check_read_rights(path)
 
         # Fetch data
-        manifest = await self._get_manifest_from_path(path)
-        return manifest.to_stats()
+        manifest, confined = await self._get_manifest_from_path(path)
+        stats = manifest.to_stats()
```

Comment:
so `to_stats()` is no longer all the stats...

Could it be a trouble for `FileTransactions.fd_info()` (the other place where this `to_stats()` is used) ?

(I would prefer not to have this `to_stats()` method given it is some kind of arbitrary conversion that depend of who is needed them, but I understant it's not that easy here given we have two places that need the stats... and now maybe the-same-stats-but-not-100%-the-same...)

Reply:
b89946e15d95d68291748803f1033db8abc61d05